confidential-datahub/hub/src/hub.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2023 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use async_trait::async_trait;
+use base64::{engine::general_purpose::STANDARD, Engine};
+use secret::secret::Secret;
+
+use crate::{DataHub, Error, Result};
+
+pub struct Hub {}
+
+#[async_trait]
+impl DataHub for Hub {
+    async fn unseal_secret(&self, secret: Vec<u8>) -> Result<Vec<u8>> {
+        // TODO: verify the jws signature using the key specified by `kid`
+        // in header. Here we directly get the JWS body
```

Comment:
This TODO makes sense, but is this a typo? Should it be, "Here we directly get the JWT body"?

Reply:
Ah, ok. My idea is the claims included in a JWS. From RFC it should be called `payload` https://www.rfc-editor.org/rfc/rfc7515.html#section-3. Let me fix this.